### DISCUSSION TITLE
We hope to support a larger, more accurate, and more powerful intelligent search model/希望支持规模更大，精准度更高，更强的智能搜索模型

### DISCUSSION THREAD

**Discussion by shenyan-008** [immich-app/immich#13445] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

We hope to support a larger, more accurate, and more powerful intelligent search model/希望支持规模更大，精准度更高，更强的智能搜索模型

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by alextran1502** [comment]
Have you checked the list of available models? https://huggingface.co/immich-app

**Reply by shenyan-008** [reply]
I have tried various Multilingual CLIPs, but I still have problems with intelligent search for images, or I want to search for images with specific features or words but cannot find them. I can be sure that there are relevant images in the gallery, but I cannot search them in Chinese or English./我尝试过了各个Multilingual CLIP，但还是对于图像的智能搜索还是有问题，或者是想要搜索指定特征或者词语的图像却没有搜索到，我可以确定图库中存在相关图片，使用中文或者英文都无法搜索出来

**Reply by shenyan-008** [reply]
https://huggingface.co/immich-app/XLM-Roberta-Large-Vit-B-16Plus  This is the model currently used

**Reply by shenyan-008** [reply]
Can the use of the model be more customized? There are many relatively good multimodal large language models now./能否让模型的使用更加自定义，现在有不少的多模态大语言模型相对不错

**Reply by shenyan-008** [reply]
It would be best if it could also support OCR of images and recognize text in images. Currently, there is no simple and usable software or project on the market./最好还能支持图片的OCR，识别图片的文字，现在市面上还未找到简单可用的软件或者项目

**Reply by alextran1502** [reply]
> Can the use of the model be more customized? There are many relatively good multimodal large language models now.

Potentially 

> It would be best if it could also support OCR of images and recognize text in images. 

Yeah we plan to have this in the future

**Reply by shenyan-008** [reply]
That's awesome.

**Reply by shenyan-008** [reply]
Regarding the purchase of licenses, can you provide channels for mainland users to purchase licenses, such as Alipay, WeChat

**Reply by mertalev** [reply]
We've talked about WeChat and will likely support it. I'm not sure about AliPay.

**Comment by mertalev** [comment]
I've been looking at some of the larger models like the InternVL2 series. They should be a big step up from the current models, especially in OCR and multilingual capabilities. But we also intend to add proper OCR with a dedicated model.

**Reply by shenyan-008** [reply]
Here I recommend two multimodal large language models that I think are good.
https://huggingface.co/OpenGVLab/InternVL2-26B-AWQ
https://huggingface.co/Qwen/Qwen2-VL-7B-Instruct-AWQ
